docs/src/components/productMaterial/MaterialTemplates.tsx

```diff
@@ -0,0 +1,314 @@
+        src: {
+          light: '/static/branding/store-templates/template-5light.jpg',
+          dark: '/static/branding/store-templates/template-5dark.jpg',
+        },
+        href: 'https://mui.com/store/items/berry-react-material-admin/',
+      },
+      {
+        name: 'Mira Pro - React Material Admin Dashboard',
+        src: {
+          light: '/static/branding/store-templates/template-3light.jpg',
+          dark: '/static/branding/store-templates/template-3dark.jpg',
+        },
+        href: 'https://mui.com/store/items/mira-pro-react-material-admin-dashboard/',
+      },
+    ],
+    [DEMOS[1]]: [
+      {
+        name: 'theFront - Multipurpose Template + UI Kit',
+        src: {
+          light: '/static/branding/store-templates/template-2light.jpg',
+          dark: '/static/branding/store-templates/template-2dark.jpg',
+        },
+        href: 'https://mui.com/store/items/the-front-landing-page/',
+      },
+      {
+        name: 'Webbee - Multipurpose landing page UI Kit',
+        src: {
+          light: '/static/branding/store-templates/template-6light.jpg',
+          dark: '/static/branding/store-templates/template-6dark.jpg',
+        },
+        href: 'https://mui.com/store/items/webbee-landing-page/',
+      },
+    ],
+    [DEMOS[2]]: [
+      {
+        name: 'Bazar Pro - Multipurpose React Ecommerce Template',
+        src: {
+          light: '/static/branding/store-templates/template-bazar-light.jpg',
+          dark: '/static/branding/store-templates/template-bazar-dark.jpg',
+        },
+        href: 'https://mui.com/store/items/bazar-pro-react-ecommerce-template/',
+      },
+    ],
+  };
+
+  const templates = TEMPLATES[demo];
+
+  return (
+    <Section bg="gradient">
+      <Box sx={{ maxWidth: 550, m: 'auto' }}>
+        <SectionHeadline
+          alwaysCenter
+          overline="Templates"
+          title={
+            <Typography variant="h2">
+              The right template for your
+              <br /> <GradientText>specific use case</GradientText>
+            </Typography>
+          }
+          description="A collection of 4.5 average rating templates, for multiple use cases, all powered by Material UI components and carefully curated by MUI's team.
+              "
+        />
+      </Box>
+      <Group rowLayout desktopColumns={2} sx={{ mt: 3 }}>
+        {DEMOS.map((name) => (
+          <Highlighter
+            key={name}
+            selected={name === demo}
+            onClick={() => {
+              setDemo(name);
+              setTemplateIndex(0);
+            }}
+          >
+            <Item
+              icon={React.cloneElement(icons[name], name === demo ? { color: 'primary' } : {})}
+              title={name}
+            />
+          </Highlighter>
+        ))}
+        <More
+          component={Link}
+          href="https://mui.com/store/?utm_source=marketing&utm_medium=referral&utm_campaign=templates-cta2#populars"
```

Comment:
I guess so we can distinguish https://mui.com/templates/ and https://mui.com/material-ui/ from the analytics.

```suggestion
          href="https://mui.com/store/?utm_source=marketing&utm_medium=referral&utm_campaign=material-templates-cta2#populars"
```

x2

Reply:
Ah, something I was always wondering about https://momenticmarketing.com/blog/sessions-ga4-vs-universal-analytics, this changed:

<img width="816" alt="Screenshot 2023-08-24 at 00 08 08" src="https://github.com/mui/material-ui/assets/3165635/d3047bd0-08ec-46aa-aebd-005952fefbc4">

Reply:
To clarify then, does your suggestion still stands, given this discovery?